pkgs/tools/admin/wander/default.nix

```diff
@@ -1,17 +1,33 @@
-{ buildGoModule, fetchFromGitHub, lib }:
+{ buildGoModule, fetchFromGitHub, installShellFiles, lib }:
 
 buildGoModule rec {
   pname = "wander";
-  version = "0.7.0";
+  version = "0.8.0";
+
+  # wander embeds commitSHA into version information
+  commitSHA = "94e61b57f353b7b28abf9c9d088319a2d98ecbfa";
 
   src = fetchFromGitHub {
     owner = "robinovitch61";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-aQqJDUDYHoUZ6ixnY3lmFOx29QpRRke5XHFIpsA+Bnw=";
+
+    sha256 = "sha256-G/TrfnmEyomdUCN5nUS9v5iqeUzgZzMLUZnfroQLZuk=";
   };
 
-  vendorSha256 = "sha256-T+URnRLumXFz48go9TN0Wha99T03OWGfDK7cQ+zKeRI=";
+  vendorSha256 = "sha256-iTaZ5/0UrLJ3JE3FwQpvjKKrhqklG4n1WFTJhWfj/rI=";
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  ldflags = [ "-X github.com/robinovitch61/wander/cmd.Version=v${version}"
+              "-X github.com/robinovitch61/wander/cmd.CommitSHA=${commitSHA}" ];
```

Comment:
```suggestion
  ldflags = [ "-X github.com/robinovitch61/wander/cmd.Version=v${version}" ];
```

Reply:
Also feel free (optional) to add a version test. To future proof for changes in upstream.

Reply:
Notice, `ldflags` is configure phase. `nativeBuildInputs` is buildPhase.
configure phase comes before build phase.